frontend/src/components/Avatar/Avatar.tsx

```diff
@@ -3,12 +3,10 @@ import React, { useMemo } from 'react'
 import ReactNiceAvatar, { genConfig } from 'react-nice-avatar'
 
 import { generateRandomColor } from '../../util/color'
-import {
-	userAvatar,
-	userAvatarBorder,
-	userAvatarText,
-	userAvatarWrapper,
-} from './Avatar.module.scss'
+import styles from './Avatar.module.scss'
+
+const { userAvatar, userAvatarBorder, userAvatarText, userAvatarWrapper } =
+	styles
```

Comment:
The esbuild plugin I'm using for generating js modules for scss doesn't seem to support named exports. Luckily there's only a handful of named export usages across the codebase, so I've manually refactored them to use the default export.